tests/qgis-projects/tests/dataviz.qgs.cfg

```diff
@@ -1,10 +1,12 @@
 {
     "metadata": {
-        "qgis_desktop_version": 32212,
-        "lizmap_plugin_version_str": "3.9.1-alpha",
-        "lizmap_plugin_version": 30901,
-        "lizmap_web_client_target_version": 30500,
-        "lizmap_web_client_target_status": "Stable",
+        "qgis_desktop_version": 32214,
+        "lizmap_plugin_version_str": "3.14.1-alpha",
+        "lizmap_plugin_version": 31401,
+        "lizmap_web_client_target_version": 30700,
+        "lizmap_web_client_target_status": "Dev",
+        "instance_target_url": "https://demo.snap.lizmap.com/lizmap_3_7/",
```

Comment:
Maybe a bug on my side or not, but I was wondering if it's normal you have some blue in this new panel D&D when looking at your screenshot in the PR description.

Reply:
OK, it was just for me if you were hitting a bug of mine, that's it